### ISSUE TITLE
[DUALBOOT]Internal Windows partitions mounted in /mnt not showing in side panel

### ISSUE THREAD

**Issue by nonokh** [mate-desktop/caja#1325]
#### Expected behaviour
Internal Windows partitions in side panel

#### Actual behaviour
No NTFS windows partitions are shown in the windows side panel

#### Linux Distribution
Linux parrot 5.2.0-2parrot1-amd64 #1 SMP Debian 5.2.9-2parrot1 (2019-08-25) x86_64 GNU/Linux

**Comment by raveit65** [comment]
How do you mount you ntfs partition?
I am using /etc/fstab to mount my win10 partition.
```
UUID=xxxxxxxxxxxxxxxx /media/Win10 ntfs defaults,uid=1000,gid=1000,noauto 0 0
```
Works well here in fedora 30/31 with kernel-5.3.14-200.fc30.x86_64.
![Bildschirmfoto zu 2019-12-10 22-08-31](https://user-images.githubusercontent.com/961604/70569153-acb96980-1b99-11ea-88ca-0b6813b1a0bd.png)

**Comment by nonokh** [comment]
It gets mounted automatically but it doesn't show up in the side panel.
The system is able to see the partitions (shows up in fdisk and also it's shows mounted in /mnt in cat /proc/mounts) , but for some reason it isn't showing in Caja

**Comment by lukefromdc** [comment]
Not totally sure but I think only mounts.seen by GVFS show up in the sidebar.
At leasr in Debian I did not see a module named."ntfs" in gvfs-backends but
not sure if that means no support or a module with another name supports it.

I have in the past seen crashes of something other than Caja (presuming GVFS)
hide all the mounts in the sidebar.

**Comment by raveit65** [comment]
> Also, any reason why you mounted them in /media ?

No, i did it because i did it always like this under linux. You can set up your mount point everywhere in the filesystem.

**Comment by yetist** [comment]
Please try to use x-gvfs-XXX option on /etc/fstab like this:
```
/dev/disk/by-uuid/4CAE8E5B5AF47502   /Movies  auto   defaults,x-gvfs-show,x-gvfs-name=My%20Movies,x-gvfs-icon=folder-videos,x-gvfs-symbolic-icon=folder-videos-symbolic  0 0
```

**Comment by nonokh** [comment]
i asked because going through some issues here in Caja i found [https://github.com/mate-desktop/caja/issues/271#issuecomment-37775109](url)

it is kinda old post but tl;dr is it does show up on side panel if u mount in /media but doesnt if you mount in /mnt (which can be overridden by _x-gvfs-show_ in fstab) which is weird becuase technically you can mount it literally anywhere  and Caja should be showing it afaik.

**Comment by nonokh** [comment]
> Please try to use x-gvfs-XXX option on /etc/fstab like this:
> 
> ```
> /dev/disk/by-uuid/4CAE8E5B5AF47502   /Movies  auto   defaults,x-gvfs-show,x-gvfs-name=My%20Movies,x-gvfs-icon=folder-videos,x-gvfs-symbolic-icon=folder-videos-symbolic  0 0
> ```

can you explain what the various x-gvfs options means ?

**Comment by nonokh** [comment]
> > Also, any reason why you mounted them in /media ?
> 
> No, i did it because i did it always like this under linux. You can set up your mount point everywhere in the filesystem.

i asked because going through some issues here in Caja i found https://github.com/mate-desktop/caja/issues/271#issuecomment-37775109

it is kinda old post but tl;dr is it does show up on side panel if u mount in /media but doesnt if you mount in /mnt (which can be overridden by x-gvfs-show in fstab) which is weird becuase technically you can mount it literally anywhere and Caja should be showing it afaik.

**Comment by raveit65** [comment]
> > > Also, any reason why you mounted them in /media ?
> >
> > No, i did it because i did it always like this under linux. You can set up your mount point everywhere in the filesystem.

> i asked because going through some issues here in Caja i found https://github.com/mate-desktop/caja/issues/271#issuecomment-37775109

I am using /mnt dir only for doing quick mounts from terminal to avoid typing long named mount points.
For constant mounts from internal disks/file-servers/etc. i am using always /media.
